helix-core/src/main/java/org/apache/helix/controller/stages/MessageOutput.java

```diff
@@ -36,6 +37,10 @@ public MessageOutput() {
 
   }
 
+  public Set<String> resourceSet() {
```

Comment:
As commented below, I think this method is not necessary.

Reply:
usage is like this:

 `clusterStatusMonitor.setResourceRebalanceStates(output.resourceSet(),
          ResourceMonitor.RebalanceStatus.NORMAL);`

Reply:
removed for now as we will have another PR for monitoring.